core/Updates/2.0-a12.php

```diff
@@ -30,7 +41,7 @@ public function getMigrationQueries(Updater $updater)
             try {
                 $rows = Db::fetchOne("SELECT COUNT(*) FROM $tableName");
                 if ($rows == 0) {
-                    $result["DROP TABLE $tableName"] = false;
+                    $result[] = $this->migration->db->dropTable($table);
```

Comment:
just fyi: $table is unprefixed and correct, $tableName is prefixed. Got confused when reviewing it :)